src/compas_fab/robots/robot.py

```diff
@@ -83,12 +89,15 @@ def from_resource_path(cls, directory, client=None):
         The directory must contain a .urdf, a .srdf file and a directory with
         the robot's geometry as indicated in the urdf file.
         """
-        urdf_importer = UrdfImporter.from_robot_resource_path(directory)
+        urdf_importer = RosFileServerLoader.from_robot_resource_path(directory)
         urdf_file = urdf_importer.urdf_filename
+        
         srdf_file = urdf_importer.srdf_filename
         urdf_model = compas.robots.model.Robot.from_urdf_file(urdf_file)
         srdf_model = RobotSemantics.from_srdf_file(srdf_file, urdf_model)
-        return cls(urdf_model, srdf_model, client)
+        print("urdf_model", urdf_model)
+        print("srdf_model", srdf_model)
```

Comment:
Left over `print` debug statements. If intended to stay, turn them into log statements instead (but probably too verbose).